### DISCUSSION TITLE
Ability to set when the workday starts

### DISCUSSION THREAD

**Discussion by 7omas** [cpvalente/ontime#1200] [Ideas]
In many festival productions the workday is shifted, and most importantly, can end long after midnight has passed.

In this scenario it would be very helpful to be able to see the full timeline and event schedule until the last event of the workday takes place.

Suggestion: Add a, per project, setting for when the new workday starts.

**Comment by cpvalente** [comment]
Hi @7omas , thank you for taking the time to make a suggestion.

I will confess that I got a slight chill on the spine when i read the title of the suggestion, the concept of days does not exist in Ontime and adding it would significantly complicate the application.

However I would like to understand the request here, maybe it is not as destructive as I fear 😅 .

> In this scenario it would be very helpful to be able to see the full timeline and event schedule until the last event of the workday takes place.

The word timeline got a bit tainted with the new timeline view. When you say full timeline, are you mentioning that the timeline view should show a days worth of work?

The only representation of a day are the indicators in the /editor rundown. Nowhere else do we separate events by blocks of time (like days)

Can you please clarify?

**Reply by 7omas** [reply]
Yes I was hoping to see the full workday in the timeline view but also that when activating roll, it would start counting from the start of the workday and not from 00:00.

I currently add the post midnight events but mark them as skip until the time actually flips over midnight. Otherwise I sometimes get a counter that will start counting to the first event after midnight instead of running the event that matches the wall time. (might have been fixed with 3.5.0, have not fully tested it yet)

I made this suggestion this way specifically since Ontime does not include dates (and is not planned to as far as I understand).

**Reply by 7omas** [reply]
Follow up to @cpvalente 

So I saw the release of the 3.5.1evaluate and started testing all the projects I had during the festival summer to see if it was now able to handle the schedule in a predictable way without needing to skip events after midnight.

In my initial testing everything seemed to behave as expected and the need to create a workday setting is probably not be needed.
